go.mod

```diff
@@ -9,15 +11,16 @@ require (
 	github.com/onsi/ginkgo/v2 v2.23.0
 	github.com/onsi/gomega v1.36.2
 	github.com/stretchr/testify v1.10.0
-	k8s.io/apimachinery v0.32.1
-	k8s.io/client-go v0.32.1
-	sigs.k8s.io/controller-runtime v0.20.3
+	k8s.io/apimachinery v0.33.0
+	k8s.io/client-go v0.33.0
+	k8s.io/kubectl v0.33.0
+	k8s.io/kubernetes v1.33.0
```

Comment:
Yea this is a concering import.

I can't really explain why we should not import from Kubernetes but I know that I would discourage this on any repo.

I don't know if scheduler framework is something we can use as a library.

The dependency chain on this is really concerning.

Reply:
```$ go mod tidy 
go: finding module for package k8s.io/kubernetes/pkg/scheduler/framework
go: downloading k8s.io/kubernetes v1.33.1
go: found k8s.io/kubernetes/pkg/scheduler/framework in k8s.io/kubernetes v1.33.1
```

it is automatically imported, I am not explicitly adding it.

Reply:
@kannon92 how easy it is to locally implement something like NodeInfo from the scheduler framework so we can remove this dep if possible.

Reply:
no longer depend on importing `k8s.io/kubernetes`